### ISSUE TITLE
[Bug]: SASS errors when upgrading to NextJS 15

### ISSUE THREAD

**Issue by rodrigofeijao** [carbon-design-system/carbon#18271]
### Package

@carbon/react

### Package version

1.72.0

### React version

18

### Description

While trying to upgrade my NextJS to v15, I've noticed a lot of errors in my console. It seems to be directly connected with some SASS functions. Anyone else experienced the same?

![Image](https://github.com/user-attachments/assets/de91871c-f8a6-4724-aa63-dd2062dc2b99)

### Reproduction/example

https://stackblitz.com/github/carbon-design-system/sandboxes/tree/main/react/latest/react-19-vite?file=package.json&preset=node=

### Steps to reproduce

* upgrade Nextjs 14 project to 15.

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by smith558** [comment]
Yes, seeing the same. Next 15 requires minimum React 19, which Carbon is not yet compatible with, so it may be due to that.

**Comment by tay1orjones** [comment]
We expanded the peer dep ranges for react to include v19. If you're using v19 be sure to install react-is. It moved to be a peer dep to prevent issues with the tabs not rendering. Using react 19 will require react-is v19.

Sass warnings like that should be non blocking. If compilation actually fails, the warnings can sometimes be helpful and provide insight into why.

There's a known issue with sass compilation if you're using Turbopack for `next dev`. https://github.com/vercel/next.js/issues/73103 You'll have to remove `--turbopack` from the `dev` command to get it working.
